server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -449,6 +447,12 @@ public static void main(String[] args) throws Exception {
       log.info("SASL is not enabled, delegation tokens will not be available");
       delegationTokensAvailable = false;
     }
+    this.timeToCacheRecoveryWalExistence =
+        aconf.getTimeInMillis(Property.MANAGER_RECOVERY_WAL_EXISTENCE_CACHE_TIME);
+  }
+
+  public long getWaitTimeBetweenScans() {
+    return this.getConfiguration().getTimeInMillis(Property.MANAGER_TABLET_GROUP_WATCHER_INTERVAL);
```

Comment:
Since this is now only used once, and the implementation is trivial, this should now be inline'd, rather than add a new API method to Manager.